src/main/java/uk/gov/hmcts/darts/authorisation/component/impl/TranscriptionIdControllerAuthorisationImpl.java

```diff
@@ -32,18 +32,14 @@ public ContextIdEnum getContextId() {
         return TRANSCRIPTION_ID;
     }
 
+    public String getEntityIdParam() {
```

Comment:
Are you missing `@Override` on this method?

Reply:
No, its not in the interface